This patch adds support for the OpenSSL SSL_CERT_FILE environment variable.
GIT_SSL_CAINFO still takes precedence.

--- git-orig/http.c.orig	2014-11-25 23:27:56.000000000 +0100
+++ git-orig/http.c	2014-11-25 23:28:48.000000000 +0100
@@ -433,6 +433,7 @@
 #if LIBCURL_VERSION_NUM >= 0x070908
 	set_from_env(&ssl_capath, "GIT_SSL_CAPATH");
 #endif
+	set_from_env(&ssl_cainfo, "SSL_CERT_FILE");
 	set_from_env(&ssl_cainfo, "GIT_SSL_CAINFO");
 
 	set_from_env(&user_agent, "GIT_HTTP_USER_AGENT");
